src/sparseml/pytorch/torchvision/utils.py

```diff
@@ -210,6 +213,7 @@ def accuracy(output, target, topk=(1,)):
         for k in topk:
             correct_k = correct[:k].flatten().sum(dtype=torch.float32)
             res.append(correct_k * (100.0 / batch_size))
+            res.append(correct_k)
```

Comment:
Is this understanding of the changes correct?

Before:
```
self.total += value * n
```
where value = correct_k * 100 / batch_size, and n = batch size,
so `correct_k * 100` would be added to total.

Now `correct_k` is being added to the total.

So since we are removing the factor of 100, that is why you added it back to the global_avg function?

Reply:
Yes, essentially! By doing this, we avoid inconsistencies with floating point due to extra multiplication and divisions.